From: Michal Cieslakiewicz <michal.cieslakiewicz@wp.pl>
Date: Thu, 21 Apr 2016 23:00:54 +0200
Subject: [PATCH] mac80211: ath9k: add GPIO support for AR9280 chip

Enable access to GPIO on Atheros wireless chip AR9280.
Support for 9280 is added to existing 9285/9287 subsystem
because these 3 chips differ only in number of GPIO pins.

Signed-off-by: Michal Cieslakiewicz <michal.cieslakiewicz@wp.pl>
---
--- a/drivers/net/wireless/ath/ath9k/gpio.c
+++ b/drivers/net/wireless/ath/ath9k/gpio.c
@@ -88,11 +88,13 @@ static void ath9k_register_gpio_chip(str
 	struct ath9k_gpio_chip *gc;
 	u16 ng;
 
-	/* for now only AR9285 and AR9287 are recognized */
+	/* supported chips are AR9280, AR9285 and AR9287 */
 	if (AR_SREV_9287(sc->sc_ah))
 		ng = AR9287_NUM_GPIO;
 	else if (AR_SREV_9285(sc->sc_ah))
 		ng = AR9285_NUM_GPIO;
+	else if (AR_SREV_9280(sc->sc_ah))
+		ng = AR9280_NUM_GPIO;
 	else
 		return;
 
